dear_petition/petition/export/forms.py

```diff
@@ -23,8 +23,8 @@ def disposition_code(self, offense):
     def get_ordered_offense_records(self):
         qs = self.petition.offense_records.select_related("offense__ciprs_record")
         return qs.order_by(
-            "offense__ciprs_record__offense_date",
             "offense__ciprs_record__file_no",
+            "offense__ciprs_record__offense_date",
```

Comment:
Since ``file_no`` will sort alphanumeric, ``99CR------`` will sort after ``02CR------``, which will make the 1999 record show after the 2002 record. I think we could use ``CASE`` and ``WHEN`` to annotate the queryset with something like:
* ``year_sort=0`` WHERE offense_date < 2000
* ``year_sort=1`` WHERE offience_date >= 2000

If we sort by ``year_sort`` first, I think it'll work. 
* e.g. ``order_by("year_sort", "offense__ciprs_record__file_no")``